_includes/scenarios/integrate/create-attribute-for-peer.md

```diff
@@ -1,15 +1,237 @@
+{: .notice--info}
 
-To avoid misunderstandings and conflicts, usually it is better to use ProposeAttributeRequestItems for changing data of the peer, so that the peer has the option of overruling the Attribute to be saved.
+<!--- TODO: Insert Link to "Propose attribute to peer" guide --->
 
-# Examples
+We will now explain how a Connector, hereinafter referred to as the Sender, can create an Attribute for another Connector, the so-called Recipient. Since understanding this creation process requires knowledge about [Requests]({% link _docs_integrate/data-model-overview.md %}#request) and how to use them in general, you should take a look at our [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}) before continuing reading this guide.
 
-- An organization onboards an already known account to enmeshed, i.e. it knows the peer and thus could create/propose attribute to the enmeshed Identity
-- An organization would like to submit something to the peer, which is reuseable, e.g. a certificate which the peer can share to others
+Please note that the general procedure is the same if the Connector wants to create an Attribute for an App user instead of another Connector. For reasons of clarity, this guide focuses on the creation process with two Connectors.
+{: .notice--info}
+
+## Request for creating Attributes
+
+The Sender wants to create an Attribute for the Recipient. To do this, the Sender must first create a suitable Request, which it can then send to the Recipient. In the following subsections, we describe the general appearance of a Request for creating Attributes.
+
+### Role of CreateAttributeRequestItem
+
+For requesting the creation of a single Attribute for the Recipient, a single RequestItem of type [CreateAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#createattributerequestitem) must be inserted into the `items` property of the [Request]({% link _docs_integrate/data-model-overview.md %}#request). It is possible to request the creation of an IdentityAttribute or a RelationshipAttribute, which must be inserted into the `attribute` property of the CreateAttributeRequestItem. Depending on whether an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) or a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) is to be created for the Recipient, the Sender has a different number of input options when defining the prospective `owner` of the Attribute. More details on the various input options when creating a Request for creating Attributes and the corresponding application scenarios can be found in the table of the [Combinations and usage scenarios of the CreateAttributeRequestItem]({% link _docs_integrate/create-attribute-for-peer.md %}#combinations-and-usage-scenarios-of-createattributerequestitem).
```

Comment:
Link the first occurences of IdentityAttribute and RelationshipAttribute (in the sentence before you currently do it).